### PR TITLE
Remove usage of global `Em`

### PR DESCRIPTION
**Pull request by HeroicEric**
Adds missing imports of `Ember` and removes usage of global `Em`

### CODE REVIEW

**Comment by rwjblue** [comment]
Can you remove from [.jshintrc](https://github.com/ember-cli/ember-twiddle/blob/master/.jshintrc#L7) also?

**Comment by joostdevries** [comment]
@HeroicEric could you merge master/rebase.

**Comment by HeroicEric** [comment]
@joostdevries updated

File: app/models/gist-file.js [hunk 1]
```diff
@@ -10,7 +11,7 @@ export default DS.Model.extend({
   /*
     Replace dots with slashes, Gists can't have directories
    */
-  filePath: Em.computed('fileName', function(key, value) {
```
Comment (stefanpenner) [inline app/models/gist-file.js]: can we instead do

```js
const { computed } = Ember
```

this will be nice once we cut over to the module API.

Comment (HeroicEric) [inline app/models/gist-file.js]: For sure. Didn't want to start making style decisions but I'm :+1: if you all are

Comment (stefanpenner) [inline app/models/gist-file.js]: :+1: from me

Comment (HeroicEric) [inline app/models/gist-file.js]: @stefanpenner Updated that one and a few more that I came across

**Comment by stefanpenner** [comment]
nice thanks!

**Pull request merged**